### PR TITLE
create internal/config/net test

### PR DESCRIPTION
**Pull request by vankichi**
Signed-off-by: vankichi <kyukawa315@gmail.com>

### Description:

I created the test for `internal/config/net.go`

grammar check has passed

### Environment:

- Go Version: 1.16
- Docker Version: 19.03.8
- Kubernetes Version: 1.18.2
- NGT Version: 1.12.3

### Types of changes:

- [ ] Bug fix [type/bug]
- [ ] New feature [type/feature]
- [x] Add tests [type/test]
- [ ] Security related changes [type/security]
- [ ] Add documents [type/documentation]
- [ ] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [ ] Update CI [type/ci]

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### Checklist:

- [x] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [x] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [ ] I have added tests and benchmarks to cover my changes.
- [x] I have ensured all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: internal/config/net.go [hunk 1]
```diff
@@ -23,27 +23,30 @@ import (
 	"github.com/vdaas/vald/internal/tls"
 )
 
-// Net represent the network configuration tcp, udp, unix socket.
+// Net represents the network configuration tcp, udp, unix socket.
```
Comment (hlts2) [inline internal/config/net.go:26]: sorry if I made a misstake 🙇 
```suggestion
// Net represents the network configuration tcp, udp, unix domain socket.
```

**Review by hlts2 — APPROVED** [review]
LGTMed

File: internal/config/net_test.go [hunk 2]
```diff
@@ -253,77 +373,190 @@ func TestNet_Bind(t *testing.T) {
...
+		},
+		{
+			name: "return flg when socketOpts.ReuseAddr true and others are false",
+			fields: fields{
+				socketOpts: &SocketOption{
+					ReuseAddr: true,
+				},
+			},
+			want: want{
+				want: 0x2,
```
Comment (kevindiu) [inline internal/config/net_test.go:434]: the flag is os dependent variable, please use the flag defined in the control package.
Please also fix another test cases

Comment (vankichi) [reply internal/config/net_test.go:434]: thx for the review. could you please more detail about what you want to say?
I used the `uint` value for the checking output value.
and here is the definition.
https://github.com/vdaas/vald/blob/4bf65f85dcaa6c28f7b755b2a1410322f86ccbbf/internal/net/control/control.go#L46-L56
I'm not sure `the flag is os dependent variable`.

Comment (kevindiu) [reply internal/config/net_test.go:434]: sorry I misread the implementation. there are no problem in the current test implementation.

**Review by kevindiu — APPROVED** [review]
LGTM

**Pull request merged**